pkg/aci/aci.go

```diff
@@ -107,8 +110,18 @@ func (aw *imageArchiveWriter) Close() error {
 // NewBasicACI creates a new ACI in the given directory with the given name.
 // Used for testing.
 func NewBasicACI(dir string, name string) (*os.File, error) {
-	manifest := fmt.Sprintf(`{"acKind":"ImageManifest","acVersion":"0.8.9","name":"%s"}`, name)
-	return NewACI(dir, manifest, nil)
+	manifest := schema.ImageManifest{
+		ACKind:    schema.ImageManifestKind,
+		ACVersion: SpecVersion,
```

Comment:
Appcs version labels use to be updated via [a script](https://github.com/coreos/rkt/blob/master/scripts/bump-appc-spec-version). Can we turn this back into a string and use `NewSemVer()`?